.changes/unreleased/Bugfix-20240124-103647.yaml

```diff
@@ -0,0 +1,3 @@
+kind: Bugfix
+body: fix updates to owner and description fields, can assign alias not just id
```

Comment:
```suggestion
body: on property_assignment resources the fields owner and definition can use alias not just id, fix an infinite plan bug because we store the wrong thing into the terraform state 
```